sdk/tests/iot/hub/test_az_iot_hub_client_c2d.c

```diff
@@ -211,12 +211,34 @@ static void _log_listener(az_log_classification classification, az_span message)
   }
 }
 
+static bool _log_listener_should_write_MQTT(az_log_classification classification)
```

Comment:
```suggestion
static bool _should_write_everything(az_log_classification classification)
```

Reply:
Why `everything`? That's not what this is doing. It is only returning true for `AZ_LOG_MQTT_RECEIVED_PAYLOAD`

Reply:
I think github may have misplaced the comment, because the function I am looking at would return true for any `mqtt`: for `AZ_LOG_MQTT_RECEIVED_TOPIC`, and for `AZ_LOG_MQTT_RECEIVED_PAYLOAD`.

Reply:
I wouldn't name that as `everything` either.

Reply:
`_should_write_any_mqtt`? But on the other hand, you call it "everything" in other places, such as in the context of core tests you only include AZ_LOG_HTTP ones, but call it everything.

Reply:
I call it everything there because it returns true for all `core` log classifications.